### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/path-operation-configuration.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/path-operation-configuration.md to Japanese.
Please review tutorial/path-operation-configuration.md.

~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.
I will fix dependency after #1571 & #1656 are merged.~

### CODE REVIEW

File: docs/ja/docs/tutorial/path-operation-configuration.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+```Python hl_lines="16"
+{!../../../docs_src/path_operation_configuration/tutorial006.py!}
+```
+
+対話的ドキュメントでは非推奨と明記されます:
+
+<img src="https://fastapi.tiangolo.com/img/tutorial/path-operation-configuration/image04.png">
+
+非推奨である場合とそうでない場合の*path operations*がどのように見えるかを確認してください:
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/path-operation-configuration.md:91]: *path operations*が非推奨である場合とそうでない場合でどのように見えるかを確認してください:

**Review by tokusumi — APPROVED** [review]
approved this.
I only have some proposal (no request change). please refer if you find better translation.

**Comment by SwftAlpc** [comment]
@tokusumi 
Thanks for the reveiw!
I fixed some translations.
Please confirm them.

File: docs/ja/docs/tutorial/path-operation-configuration.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## タグ
+
+`tags`パラメータを`str`の`list`（通常は１つの`str`）と一緒に渡すと、*path operation*にタグを追加できます:
+
+```Python hl_lines="17 22 27"
+{!../../../docs_src/path_operation_configuration/tutorial002.py!}
+```
+
+これらはOpenAPIスキーマに追加され、自動ドキュメントのインターフェースで使用されます:
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/path-operation-configuration.md:35]:
```suggestion
これらはOpenAPIスキーマに追加され、自動ドキュメントのインターフェースで使用されます:
```
見落としてました。文字化けあります。

File: docs/ja/docs/tutorial/path-operation-configuration.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+{!../../../docs_src/path_operation_configuration/tutorial003.py!}
+```
+
+## docstringを用いた説明
+
+説明文は長くて複数行におよぶ傾向があるので、関数<abbr title="ドキュメントに使用される関数内の最初の式（変数に代入されていない）としての複数行の文字列">docstring</abbr>内に*path operation*の説明文を宣言できます。すると、**FastAPI** は説明文を読み込んでくれます。
+
+docstringに<a href="https://en.wikipedia.org/wiki/Markdown" class="external-link" target="_blank">Markdown</a>を記述すれば、正しく解釈されて表示されます。（docstringのインデントを考慮して）
+
+```Python hl_lines="19 20 21 22 23 24 25 26 27"
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/path-operation-configuration.md:53]:
```suggestion
```Python hl_lines="19-27"
```
current syntax

File: docs/ja/docs/tutorial/path-operation-configuration.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+!!! check "確認"
+    OpenAPIは*path operation*ごとにレスポンスの説明を必要としています。
+
+    そのため、それを提供しない場合は、**FastAPI** が自動的に「成功のレスポンス」を生成します。
+
+<img src="https://fastapi.tiangolo.com/img/tutorial/path-operation-configuration/image03.png">
+
+## 非推奨の*path operation*
+
+*path operation*を<abbr title="非推奨、使わない方がよい">deprecated</abbr>としてマークする必要があるが、それを削除しない場合は、`deprecated`パラメータを渡します:
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/path-operation-configuration.md:81]:
```suggestion
*path operation*を<abbr title="非推奨、使わない方がよい">deprecated</abbr>としてマークする必要があるが、それを削除しない場合は、`deprecated`パラメータを渡します:
```
文字化けあります。

File: docs/ja/docs/tutorial/path-operation-configuration.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+これらはOpenAPIスキーマに追加され、自動ドキュメントのインターフェースで使用されます:
+
+<img src="https://fastapi.tiangolo.com/img/tutorial/path-operation-configuration/image01.png">
+
+## 概要と説明
+
+`summary`と`description`を追加できます:
+
+```Python hl_lines="20 21"
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/path-operation-configuration.md:43]:
```suggestion
```Python hl_lines="20-21"
```
current syntax

File: docs/ja/docs/tutorial/path-operation-configuration.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+!!! note "技術詳細"
+    また、`from starlette import status`を使用することもできます。
+
+    **FastAPI** は開発者の利便性を考慮して、`fastapi.status`と同じ`starlette.status`を提供しています。しかし、これはStarletteから直接提供されています。
+
+## タグ
+
+`tags`パラメータを`str`の`list`（通常は１つの`str`）と一緒に渡すと、*path operation*にタグを追加できます:
+
+```Python hl_lines="17 22 27"
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/path-operation-configuration.md:31]:
```suggestion
```Python hl_lines="17  22  27"
```
current syntax

**Review by tokusumi — CHANGES_REQUESTED** [review]
I'm sorry. I found something left uncorrected.

File: docs/ja/docs/tutorial/path-operation-configuration.md (hunk 3 above)
Comment (SwftAlpc) [reply docs/ja/docs/tutorial/path-operation-configuration.md:53]: oh, I see that the writing style of hl_lines has improved while I've been away for a bit.
 I'll fix it.

File: docs/ja/docs/tutorial/path-operation-configuration.md (hunk 4 above)
Comment (SwftAlpc) [reply docs/ja/docs/tutorial/path-operation-configuration.md:81]: エディターの設定がおかしいのかローカルではこの文字化けが再現しないようです...
ご迷惑おかけいたします 🙇

**Comment by tiangolo** [comment]
📝 Docs preview for commit 99ca29389e49d1bcf775da34908ad66a9a1669cb at: https://6499c6203df37400be860887--fastapi.netlify.app

**Comment by tiangolo** [comment]
It seems like the changes requested by @tokusumi are handled, and as @SwftAlpc is one of the champions of the Japanese translations, I'll go ahead and merge this. :rocket:

**Pull request merged**